src/llama_stack_client/lib/direct/direct.py

```diff
@@ -17,15 +20,21 @@
 
 
 class LlamaStackDirectClient(LlamaStackClient):
-    def __init__(self, config: StackRunConfig, **kwargs):
+    def __init__(self, config: StackRunConfig | str, **kwargs):
```

Comment:
this is not clear to the user. Can we instead create two static methods `LlamaStackDirectClient.from_yaml()` and `LlamaStackDirectClient.from_template()` at least?

Reply:
done.